index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Enceladus
+---
+
+Enceladus is an open source project under the [ABSA Group Limited][absa] designed for transforming
+any data format into a standardized and strongly type `parquet` format and applying a number of
+conformance rules on top of said data.
+
+Enceladus laverages [Apache Spark][spark] to read and transform the data. On top of that it also
+uses [Cobrix][gh-cobrix] to be able to read COBOL data, [Spline][gh-spline] to keep track of the
+data lineage, and many other open sourrce tools.
+
+Enceladus is shipped as a UI for managing data sets and 2 spark jobs for transforming said data sets.
```

Comment:
What about the API part of Menas?

Reply:
Not supported currently and should not be mentioned before the audit.

Reply:
Fair enough